### ISSUE TITLE
Phonebook - remove text clipping on result phone numbers, email addresses

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#355]
**Describe the bug**

https://github.com/UCF/Main-Site-Theme/blob/ba14233e19e3dd253e6584628a25049d3311defd/src/scss/pages/_phonebook.scss#L29-L35

`overflow: hidden;`, `text-overflow: ellipsis;`, and `white-space: nowrap;` should be removed/adjusted from the snippet linked above to ensure that users zoomed up to 200% will not have phone/email contents clipped that users at a standard zoom level would be able to see.

**Steps to Reproduce**

n/a

**Expected Behavior**

Text should never be clipped/truncated more when users zoom their browser up to 200%.

**Possible Fix**

Remove the rules noted above; look into `overflow-wrap: break-word` to ensure long email addresses and phone numbers are able to break mid-string and not cause layout problems: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

**Your environment:**

n/a

**Additional context**

https://www.w3.org/WAI/WCAG21/Understanding/resize-text
